pvlib/iotools/psm3.py

```diff
@@ -74,50 +83,25 @@ def get_psm3(latitude, longitude, api_key, email, names='tmy', interval=60,
 
         ['1998', '1999', '2000', '2001', '2002', '2003', '2004', '2005',
          '2006', '2007', '2008', '2009', '2010', '2011', '2012', '2013',
-         '2014', '2015', '2016', '2017', 'tmy', 'tmy-2016', 'tmy-2017',
-         'tdy-2017', 'tgy-2017']
-
-    The return is a tuple with two items. The first item is a header with
-    metadata from NREL PSM3 about the record containing the following fields:
-
-    * Source
-    * Location ID
-    * City
-    * State
-    * Country
-    * Latitude
-    * Longitude
-    * Time Zone
-    * Elevation
-    * Local Time Zone
-    * Dew Point Units
-    * DHI Units
-    * DNI Units
-    * GHI Units
-    * Temperature Units
-    * Pressure Units
-    * Wind Direction Units
-    * Wind Speed
-    * Surface Albedo Units
-    * Version
-
-    The second item is a dataframe with the timeseries data downloaded.
+         '2014', '2015', '2016', '2017', '2018', 'tmy', 'tmy-2016', 'tmy-2017',
+         'tdy-2017', 'tgy-2017', 'tmy-2018', 'tdy-2018', 'tgy-2018']
```

Comment:
This list is going to be outdated in ~6 months, but I don't have a better idea for describing the values. The information on the NSRDB websites are also out of date.

Reply:
I think these values are up to date in the developer docs at least: [TMY endpoint](https://developer.nrel.gov/docs/solar/nsrdb/psm3_tmy_data_download/#request-parameters) and [Single-Year endpoint](https://developer.nrel.gov/docs/solar/nsrdb/psm3_data_download/#request-parameters).  Side note, these pages are developed on GH: https://github.com/NREL/developer.nrel.gov/commit/5c03e14815f4a58dab36dc7584585672b2060115

I just put in a request for the [TMY Description](https://nsrdb.nrel.gov/about/tmy.html) to be updated to include 2018.